### PR TITLE
Non visualization fixes and utilities

### PR DESCRIPTION
**Pull request by piotrm0**
- Reformatted a lot of comment text.
- Added some new comments for todos or unknown points.
- Added __str__ to various trulens types.
- Cleared notebook outputs.
- Added some error handling and suggestion if value cannot be extracted from fprop.
- Added another NLP baseline that swaps particular tokens.
- Added utilities for running notebooks in colab.

### CODE REVIEW

File: trulens/nn/quantities.py [hunk 1]
```diff
@@ -46,6 +44,13 @@ class QoI(AbstractBaseClass):
     output behavior that the attributions describe.
     """
 
+    def __str__(self):
+        return render_object(self, [])
+
+    # TODO: Need to give a seperate value of y at target instance here since
+    # these are values are interventions. Cannot presently define a QoI that says:
+    # logits of the predicted class for each instance.
```
Comment (rshih32) [inline trulens/nn/quantities.py:52]: are these comments tied to anything?

**Review by rshih32 — APPROVED** [review]

File: trulens/utils/colab.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Some utilities to help with colab/jupyter notebook demos.
+
+import importlib
+from pathlib import Path
+import subprocess
+import sys
+
+
+def install_if_not_installed(packages):
```
Comment (coreyhu) [inline trulens/utils/colab.py:9]: Are these used anywhere?

**Review by coreyhu — APPROVED** [review]

File: trulens/utils/colab.py (hunk 2 above)
Comment (piotrm0) [reply trulens/utils/colab.py:9]: In notebooks to be run easily in colab but I see none of them are presently in this PR. They are part of the visualizations PR. Exception to include them in this PR?

File: trulens/nn/quantities.py (hunk 1 above)
Comment (piotrm0) [reply trulens/nn/quantities.py:52]: Yes. https://github.com/truera/trulens/issues/72 and some task on the old task tracker which I cannot find now. Will make a new one and refer to it in the comment.

**Pull request merged**